java-symbolic-execution/src/main/resources/org/sonar/java/se/xproc/org.springframework.util.json

```diff
@@ -378,6 +521,28 @@
     "signature": "org.springframework.util.Assert#isTrue(ZLjava/lang/String;)V",
     "varArgs": false,
     "declaredExceptions": [],
+    "yields": [
+      {
+        "parametersConstraints": [
+          ["FALSE"],
+          []
+        ],
+        "exception": "java.lang.IllegalArgumentException"
+      },
+      {
+        "parametersConstraints": [
+          ["TRUE"],
+          []
+        ],
+        "resultIndex": -1,
+        "resultConstraint": ["TRUE"]
```

Comment:
the method is a void method (`(...)V`) so it can not return anything and having `["TRUE"]` here is wrong. It should be `null` instead.